src/dl_txt_unpack.cpp

```diff
@@ -659,7 +669,10 @@ static void dl_txt_unpack_write_member_subdata(dl_ctx_t dl_ctx, dl_txt_unpack_ct
 						uint32_t  array_count  = *(uint32_t*)(member_data + sizeof(uintptr_t));
 						const uint8_t* array = unpack_ctx->packed_instance + array_offset;
 						for( uint32_t i = 0; i < array_count; ++i )
-							dl_txt_unpack_write_subdata( dl_ctx, unpack_ctx, writer, subtype, array + i * subtype->size[DL_PTR_SIZE_HOST] );
+				        {
```

Comment:
One more

Reply:
This is my Visual Studio switching tabs to spaces in some places, and I don't know why yet. And with tabs set to 4 spaces you don't notice these. :(